web/app/themes/ppj/src/vue/FindAJob.vue

```diff
@@ -143,10 +146,15 @@
           <button class="find-a-job__pagination-direction"
                   :class="{'find-a-job__pagination-direction--enabled': (backwardEnabled == true)}"
                   @click.stop.prevent="showPreviousPage">
-            <svg x="0px" y="0px"
-                 viewBox="0 0 32.9 32.9"
-                 xml:space="preserve"
+            <svg
+                aria-labelledby="previousPageButtonTitle previousPageButtonDesc"
+                viewBox="0 0 32.9 32.9"
+                x="0px"
+                xml:space="preserve"
+                y="0px"
             >
+              <title id="previousPageButtonTitle">Previous page button</title>
+              <desc id="previousPageButtonDesc">A shape like a less than sign</desc>
```

Comment:
"An arrow pointing left" might be a more user friendly description of this image.

Reply:
Will discuss with Josh.